.github/workflows/unit_tests.yml

```diff
@@ -79,7 +82,25 @@ jobs:
       if: "matrix.os == 'ubuntu-latest'"
       run: |
         pip install -U pandas
-    - name: Run pytest
+    - name: Run pytest (Linux)
+      if: startsWith(matrix.os, 'ubuntu')
+      run: |
+        export BIDS_VALIDATOR_VERSION=`bids-validator --version`
+        echo Using bids-validator $BIDS_VALIDATOR_VERSION
+        xvfb-run python -m pytest . --cov=mne_bids mne_bids/tests/ mne_bids/commands/tests/ --cov-report=xml --cov-config=setup.cfg --verbose --ignore mne-python
```

Comment:
why do you need `xvfb` for matplotlib stuff? or is it for the code overage report?

Reply:
I removed it, we're testing with the Agg backend now.